CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- `notes` - match invalid filename chars and replace with dash (db25ec9 by hyx131)
```

Comment:
have to be careful about the hash when an overriding force push has happened (in this case due to `git commit --amend`)